book/src/csi-crds.md

```diff
@@ -0,0 +1,14 @@
+# CSI CRDs
+
+The Kubernetes CSI team created a set of [Custom Resource Definitions](https://kubernetes.io/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definitions/#create-a-customresourcedefinition) (CRDs).
+
+The CRDs can be found here: https://github.com/kubernetes/csi-api/tree/master/pkg/crd/manifests
+
+The CRDs are automatically deployed on Kubernetes via a Kubernetes Storage CRD addon https://github.com/kubernetes/kubernetes/tree/master/cluster/addons/storage-crds
```

Comment:
Should we mention that if you will need to update your cluster deployment mechanism to enable the addon and/or install the crds?

Reply:
Done.